scenes/cslib_number.txt

```diff
@@ -118,6 +118,52 @@
 *goto min_loop
 
 
+*comment MAX_STAT
+*comment ------------------------------
+*comment Returns the largest of a set of given stats.
+*comment ------------------------------
+*comment 	params:
+*comment		param_n(string): a stat (passed by name)
+*comment	returns: 
+*comment		(string): the stat with the largest value
```

Comment:
the variable name of the stat with the largest value

Reply:
done